index.html

```diff
@@ -36,6 +36,196 @@ <h1>
             <li><a href="#"><img src="./assets/victory.svg" alt="victory"></a></li>
         </ul>
     </section>
+
+    <section id="works">
+        <div class="project">
+            <div>
+                <img src="./assets/SnapshootPortfolio.svg" alt="">
+            </div>
+
+            <div class="projectDescription">
+                <div class="projectTitle">
+                    <span class="heading">Tonic</span>
+                </div>
+                <ul class="projectTools">
+                    <li class="canopy">CANOPY</li>
+                    <li><img src="./assets/Counter.svg" alt=""></li>
+                    <li>Back End Dev</li>
+                    <li><img src="./assets/Counter.svg" alt=""></li>
+                    <li>2015</li>
+                </ul>
+                <div class="projectSummary">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="projectTechnology">
+                    <span class="tag">HTML</span>
+                    <span class="tag">CSS</span>
+                    <span class="tag">JavaScript</span>
+                </div>
+                <div class="projectDemo">
+                    <button class="enabled">See project</button>
+                </div>
+            </div>
+        </div>
+
+        <div class="project">
+            <div>
+                <img src="./assets/SnapshootPortfolio1.svg" alt="">
+            </div>
+
+            <div class="projectDescription">
+                <div class="projectTitle">
+                    <span class="heading">Multi-Post
+                        Stories</span>
+                </div>
+                <ul class="projectTools">
+                    <li class="canopy">CANOPY</li>
+                    <li><img src="./assets/Counter.svg" alt=""></li>
+                    <li>Back End Dev</li>
+                    <li><img src="./assets/Counter.svg" alt=""></li>
+                    <li>2015</li>
+                </ul>
+                <div class="projectSummary">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="projectTechnology">
+                    <span class="tag">HTML</span>
+                    <span class="tag">CSS</span>
+                    <span class="tag">JavaScript</span>
+                </div>
+                <div class="projectDemo">
+                    <button class="enabled">See project</button>
+                </div>
+            </div>
+        </div>
+
+        <div class="project">
+            <div>
+                <img src="./assets/SnapshootPortfolio2.svg" alt="">
+            </div>
+
+            <div class="projectDescription">
+                <div class="projectTitle">
+                    <span class="heading">Tonic</span>
```

Comment:
- [ ] Great job removing the multiple h1 tags. But note that this is still a heading and it is best to use a semantic element. Kindly use heading tags between `h2 and h5` for all heading/titles.